src/System.ComponentModel.TypeConverter/src/System/ComponentModel/UriTypeConverter.cs

```diff
@@ -52,7 +52,7 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
 
             if (value is Uri uri)
             {
-                return new Uri(uri.OriginalString); // return new instance
+                return new Uri(uri.OriginalString, GetUriKind(uri));
```

Comment:
Any reason we don't just do `return uri;` here?

Reply:
Semantics of the converter is to return a copy.  I guess it *shouldn't* matter since Uri is immutable (I think) but my goal of this was to bring back the desktop behavior, not diverge from it in some new way.

Reply:
Ok.

Reply:
Also, it looks like Uri is not sealed, so someone may pass in a derived Uri and should get a standard Uri out, not the original derived Uri.  Also the derived Uri could be mutable.